opennms-config-model/src/main/java/org/opennms/netmgt/config/discovery/ExcludeRange.java

```diff
@@ -58,6 +59,14 @@
     @XmlElement(name = "end", required = true)
     private String m_end;
 
+    /**
+     * The monitoring location where this exclude range
+     *  will be excluded
+     */
+    @XmlAttribute(name = "location")
```

Comment:
We should add support for this field to the UI as well, since we already support configuring the exclude ranges there.